src/product/index.ts

```diff
@@ -0,0 +1,212 @@
+import api from '../../api'
+import { Product } from '../../types/Product';
```

Comment:
```suggestion
import { Product } from '../../types';
```
Export product type from the types index.ts file

Reply:
updated